### PR TITLE
[PIL-535][ND]Reduce duplicates for NFM

### PR DESCRIPTION
**Pull request by neelu-dwivedi**

### CODE REVIEW

File: src/test/resources/features/NFMNoIDFlowPages.feature [hunk 1]
```diff
@@ -1,9 +1,9 @@
 @tests @zap_accessibility
-Feature: Nominated Filing member check your answers page
+Feature: NFM NO ID journey
   As a MNE user
-  I would like to change data from change your answers page
+  I would like to enter my details via NFM No ID journey
 
-  Scenario: 1 - User navigates to check your answer page answering all the questions of UPE and NFM
+  Scenario: 1 - NFM No Id journey navigation to check your answers page and verify if data is pre populated
```
Comment (annemonica5) [inline src/test/resources/features/NFMNoIDFlowPages.feature:6]: Please validate the header and captions for all the NFM No ID journey in this scenario.

File: src/test/resources/features/NFMNoIDFlowPages.feature [hunk 2]
```diff
@@ -121,12 +123,80 @@ Feature: Nominated Filing member check your answers page
...
+    When I click on Continue button
+    Then I should see NFM error message Enter the first line of the address on the Address Line 1 Element
+    And I should see NFM error message Enter the town or city on the City Element
+    And I should see NFM error message Select a country on the Country Element
+    When I enter Address Line 1 as Address Line 1 Character Length Test1
+    And I enter Address Line 2 as Address Line 2 Character Length Test1
+    And I enter City as City Field Character Length Test Error
+    And I enter Region as Region Field Character Length Test Error
+    And I enter Postal Code as 12345678901
+    And I select country as India
```
Comment (annemonica5) [inline src/test/resources/features/NFMNoIDFlowPages.feature:157]: Postal code error validation for country UK is missed.

**Review by annemonica5 — COMMENTED** [review]
Please include the Telephone No journey navigation to check your answer and validate the back link in all the pages.

**Review by annemonica5 — APPROVED** [review]

**Review by srikanthhmrc — APPROVED** [review]
LGTM

**Pull request merged**